pkg/cmd/server/origin/master_config.go

```diff
@@ -144,9 +144,13 @@ type MasterConfig struct {
 	// APIClientCAs is used to verify client certificates presented for API auth
 	APIClientCAs *x509.CertPool
 
-	// PrivilegedLoopbackClientConfig is the client configuration used to call OpenShift APIs from system components
+	// PrivilegedLoopbackKubernetesClientConfig is the client configuration used to call Kubernetes APIs from system components
 	// To apply different access control to a system component, create a client config specifically for that component.
-	PrivilegedLoopbackClientConfig restclient.Config
+	PrivilegedLoopbackKubernetesClientConfig restclient.Config
+
+	// PrivilegedLoopbackOpenShiftClientConfig is the client configuration used to call OpenShift APIs from system components
+	// To apply different access control to a system component, create a client config specifically for that component.
+	PrivilegedLoopbackOpenShiftClientConfig restclient.Config
```

Comment:
Is this one actually different?  We no longer allow the proxy to kube configuration.